### DISCUSSION TITLE
ZIM Files

### DISCUSSION THREAD

**Discussion by emehri** [goldendict/goldendict#1581] [Q&A] [answered]
Hello,
Most of ZIM databases have troubles. GoldenDict can't Index them and does not load them at all.
For example:
https://dumps.wikimedia.org/kiwix/zim/wikipedia/wikipedia_es_all_nopic_2022-11.zim

**Comment by vedgy** [comment, marked as answer]
The download link is broken. If the bug affects the latest GoldenDict master, please create an issue (not a discussion).
